### PR TITLE
Made cart management page.

### PR DESCRIPTION
**Pull request by j24m**
Hello people made a cart management page .
below is the video for it.

https://user-images.githubusercontent.com/53934353/154847133-ed0220be-ad37-4aeb-b7f9-6597d4d79f09.mp4

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Looks great to merge! 
Clean UI and code!

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+</head>
+
+<body>
+    <div class="cart-page-container">
+        <!-- Nav bar starts here -->
+        <div class="nav-bar">
+            <a class="homepage-nav-links" href="./index.html" target="_blank">
+                <div class="homepage-nav-logo">
+                    <div>FlameKeys</div>
+                    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (pankaj-prog) [inline cart.html:23]: use semantics tags

File: cart.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    align-items: baseline;
+}
+
+.cart-card-container .bill-details-card .ul .li-price {
+    width: 40%;
+    text-align: right;
+}
+
+.cart-card-container .bill-details-card .ul li:hover {
+    transform: scale(1);
```
Comment (rahulyadav139) [inline cart.css:41]: I am not sure this transform property is coming into effect on hovering on the element.

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+.cart-card-container {
+    padding: 1rem;
+    height: 70vh;
+    display: flex;
```
Comment (pankaj-prog) [inline cart.css]: your cart content may increase than the 70vh, you may use min height

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: cart.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    justify-content: center;
+    align-items: center;
+    gap: 2rem;
+}
+
+.cart-card-container .horizontal-card{
```
Comment (rahulyadav139) [inline cart.css]: Cart is looking great. Animations are subtle and smooth. Great work.

**Review by rahulyadav139 — APPROVED** [review]

**Review by zoef-yt — APPROVED** [review]
The code looks good. Good to merge.

**Review by mehulsatardekar — APPROVED** [review]
Do the changes as others have suggested, other than that you can merge the branch

File: cart.html (hunk 1 above)
Comment (j24m) [reply cart.html:23]: This change will require me to make changes in html code of multiple files, so will make the changes later, but thanks for your suggestion :)

File: cart.css (hunk 3 above)
Comment (j24m) [reply cart.css]: got it, will do it rn.

File: cart.css (hunk 2 above)
Comment (j24m) [reply cart.css:41]: Actually I have used .ul class in my component library where I had given a hover property transform : scale(1.1), which was being inherited over here, but here I did not want a hover effect hence the scale(1).

**Pull request merged**